src/Zend/Config/Xml.php

```diff
@@ -193,7 +193,7 @@ protected function _toArray(SimpleXMLElement $xmlObject)
         $attributes   = $xmlObject->attributes();
 
         // Search for parent node values
-        if ($attributes !== null && count($attributes) > 0) {
+        if ($attributes !== null && $attributes->count() > 0) {
```

Comment:
In PHP 7.4, `SimpleXmlElement` implements `Countable` so this isn't required in PHP 7.4+, but this fix will make phpstan/psalm happy in PHP < 7.4 (and is a documented method on the `SimpleXmlElement` class: https://www.php.net/manual/en/simplexmlelement.count.php).